README.md

```diff
@@ -37,9 +37,21 @@ You can quickly start a locally running chat-ui & LLM text-generation server tha
 
 **Step 1 (Start llama.cpp server):**
 
+Install llama.cpp w/ brew (for Mac):
```

Comment:
brew also exists for linux no?

Reply:
It does, however, it only instals CPU only build, so would be quite slow. So recommended way would still be to `clone` + `make` imo.